### PR TITLE
Add configuration option for # backend workers

### LINKED ISSUE #40: Allow configurable scaling
Add a setting so the number of workers used by the python backend can be configured.
Currently the python backend is using 17 workers (`-w` flag):
https://github.com/Imageomics/Andromeda/blob/9f5489b86625d5e01debb502a8f8b31ef7a39037/andromeda/Dockerfile#L10

The suggested number of workers is based on the number of CPU available: https://docs.gunicorn.org/en/stable/design.html#how-many-workers

### PR DESCRIPTION
**Pull request by johnbradley**
Adds support for a BACKEND_WORKERS environment variable to control the number of workers used by the backend container. This controls the number of gunicorn processes. A example.env file is provided to give a safe default value for BACKEND_WORKERS. I broke out the command into a shell script within the Docker container to read the environment variable.

Fixes #40

### CODE REVIEW

**Review by egrace479 — APPROVED** [review]
LGTM

**Comment by johnbradley** [comment]
@thompsonmj When you get a chance could you review this PR? Thanks.

**Review by thompsonmj — COMMENTED** [review]
Question about selecting the number of backend workers.

File: andromeda/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ARG BACKEND_WORKERS=3
```
[the remark this answers is not in the document]
Comment (johnbradley) [reply andromeda/Dockerfile:1]: I don't think you can use a bash expression for the value of this item. I think with the changes for run.sh this line isn't necessary any more though.
```suggestion
```

**Comment by thompsonmj** [comment]
How about adding `export BACKEND_WORKERS=$((2*$(nproc)+1))` to the deployment notes just before `docker-compose`? That should make it automatically adjust if/when we change the EC2 instance type.

**Comment by thompsonmj** [comment]
Or just put that formula in the `.env`?

File: andromeda/run.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gunicorn -w $BACKEND_WORKERS -b :5000 -t 360 wsgi:app
```
Comment (johnbradley) [inline andromeda/run.sh:2]: @thompsonmj So you are suggesting this:
```suggestion
gunicorn -w ${BACKEND_WORKERS:=4} -b :5000 -t 360 wsgi:app
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ To deploy the website on a machine running docker run the following commands:
 ```bash
 git clone https://github.com/Imageomics/Andromeda.git
 cd Andromeda/
+cp example.env .env
```
Comment (thompsonmj) [inline README.md:14]: export BACKEND_WORKERS=$((2*$(nproc)+1))

**Review by thompsonmj — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (johnbradley) [reply README.md:14]: I would rather have the value set within `.env`. Otherwise this step might get lost when someone deploys new changes.

Comment (johnbradley) [reply README.md:14]: I'm going to go ahead and merge, but feel free to open an issue if you want to discuss this further.

**Pull request merged**